### PR TITLE
[plugin-web-app] Deprecate current page validation step

### PR DESCRIPTION
**Pull request by PSharuba**

### CODE REVIEW

File: vividus-tests/src/main/resources/story/integration/PageStepsTests.story [hunk 1]
```diff
@@ -31,12 +31,14 @@ Examples:
 |PAGE_LOAD_TIME       |
 
 Scenario: Verify URL components of the opened page
-Given I am on page with URL `${vividus-test-site-url}/mouseEvents.html`
+Given I am on page with URL `${vividus-test-site-url}/mouseEvents%E2%82%AC.html`
```
Comment (valfirst) [inline vividus-tests/src/main/resources/story/integration/PageStepsTests.story:34]: such page doesn't exist

**Comment by PSharuba** [comment]
Checked on Chrome, Edge and Firefox. All browsers return encoded url for current_page_url dynamic variable

File: vividus-tests/src/main/resources/story/integration/PageStepsTests.story [hunk 2]
```diff
@@ -31,12 +31,14 @@ Examples:
 |PAGE_LOAD_TIME       |
 
 Scenario: Verify URL components of the opened page
-Given I am on page with URL `${vividus-test-site-url}/mouseEvents.html`
+Given I am on page with URL `${vividus-test-site-url}/mouseEvents.html?query=%E2%82%AC`
```
Comment (valfirst) [inline vividus-tests/src/main/resources/story/integration/PageStepsTests.story:34]: do we have URL query handling at this page?

Comment (PSharuba) [reply vividus-tests/src/main/resources/story/integration/PageStepsTests.story:34]: I don't think so, It loads the same page without any changes. Can be removed if necessary

File: docs/modules/plugins/pages/plugin-web-app.adoc [hunk 3]
```diff
@@ -1056,6 +1056,19 @@ When I save number of open tabs to SCENARIO variable `count`
...
+To validate URL of the current page the following statement can be used:
+
+.Validate URL of "https://example.com/relative/url/example.html"
+[source,gherkin]
+----
+Given I am on page with URL `https://example.com/relative/url/example.html`
+Then `${current-page-url}` is equal to `https://example.com/relative/url/example.html`
+----
+
+* `${current-page-url}` is the <<Current page URL,current page URL>> dynamic variable.
```
Comment (valfirst) [inline docs/modules/plugins/pages/plugin-web-app.adoc:1070]: I believe this is already described in section of current page URL dynamic variable

File: vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/PageSteps.java [hunk 4]
```diff
@@ -203,10 +203,15 @@ public void openPageUrlInNewWindow(String pageUrl)
      * <b>URL</b> is the internet address of the current page which is located in the address bar
      * <p>
      * @param url String value of URL
+     * @deprecated Use combination of step and expression:
```
Comment (valfirst) [inline vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/PageSteps.java:206]:
```suggestion
     * @deprecated Use combination of step and dynamic variable:
```

**Review by valfirst — CHANGES_REQUESTED** [review]

File: docs/modules/plugins/pages/plugin-web-app.adoc (hunk 3 above)
Comment (PSharuba) [reply docs/modules/plugins/pages/plugin-web-app.adoc]: removed

**Review by uarlouski — APPROVED** [review]

**Review by ikalinin1 — APPROVED** [review]

**Pull request merged**